webapp/src/components/AssetPage/IconBadge/IconBadge.tsx

```diff
@@ -2,18 +2,33 @@ import React from 'react'
 import classNames from 'classnames'
 import { Props } from './IconBadge.types'
 import './IconBadge.css'
+import { Link } from 'react-router-dom'
 
-const IconBadge = ({ icon, text, onClick, className, children }: Props) => {
-  return (
-    <div className={classNames('IconBadge', className)} onClick={onClick}>
+const IconBadge = ({ icon, text, onClick, href, className, children }: Props) => {
+
+  const childrenInt = React.useMemo(() => (
```

Comment:
We could just import `useMemo` right?

```suggestion
  useMemo(() => {
```